iFixit/toolbox
# Add display inline-block to ButtonContainer

This adds a rule for the ButtonContainer to be `display: inline-block`.

This is the default for button tags but for anchor tags the default is
'inline'. We use an anchor tag as the button container when it is a link.
This means the behavior of the container is different depending on 
whether it's a link.

Changing this fixes a layout bug with the PageNavbar on wikis: https://github.com/iFixit/ifixit/issues/21066

QA
===
- Make sure the layout bug in https://github.com/iFixit/ifixit/issues/21066 is fixed
- Check a couple other places with buttons as links
